server/chat-plugins/trivia.ts

```diff
@@ -122,6 +122,22 @@ function isTriviaRoom(room: Room) {
 	return false;
 }
 
+function getTriviaGame(context: CommandContext) {
+	const room = Rooms.get('trivia');
+	if (!room) return false;
+	if (!room.game) {
+		context.errorReply(`There is no game in progress.`);
+		return false;
+	}
+	if (room.game && !room.game.gameid.includes('trivia')) {
+		context.errorReply(`There is already a game of ${room?.game.title} running.`);
+		return false;
+	}
+	if (room.game.gameid.includes('trivia')) {
+		return room.game as Trivia;
+	}
```

Comment:
This should probably be modeled way more like `getGame`? Just make sure the constructor ends with `Trivia`.